recipes/capnproto/all/patches/0009-windows-symlink-fix-0.8.0.patch

```diff
@@ -0,0 +1,20 @@
+Workaround for install step on Windows where symlinks might be not supported.
```

Comment:
Will you send this patch upstream?

Reply:
Sure, but it I decided to fix the recipe first to use in my project.

Reply:
@madebr here is the pull request to the original project: https://github.com/capnproto/capnproto/pull/1263

Reply:
Great to see this!
It's much appreciated!